daemon/web/styles/spacing.sass

```diff
@@ -52,6 +52,9 @@ $spacing-xxl: 112px
     &-xs
       padding-left: $spacing-xs
 
+    &-s
+      padding-left: $spacing-s
```

Comment:
if you are using `pad-sides-s` [here](https://github.com/ubclaunchpad/inertia/pull/399/files#diff-cc20f01c67746d3327daef444bae36c0R26), make sure you add a declaration for `padding-right` under `pad-right-s` too, otherwise you'll only get `left`